### PR TITLE
OCP-DEMO Add slots to server task deployment config

### PR DESCRIPTION
**Pull request by fax4ever**
Infinispan server 9.3 uses now `ispn-9.3` as internal slot module.

### CODE REVIEW

File: openshift/message-board/message-service/src/test/java/org/hibernate/demo/message/post/it/MessageTaskIT.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	@TargetsContainer("wildfly")
+	public static WebArchive doDeploy() {
+		return DeploymentUtil.create();
+	}
+
+	@Deployment(name = "infinispan", testable = false)
+	@TargetsContainer("infinispan")
+	public static JavaArchive getInfinispanDeployment() {
+		File file = Maven.resolver()
+			.resolve( "org.hibernate.demos.messageboard:message-server-task:jar:1.0.0-SNAPSHOT" )
```
Comment (DavideD) [inline openshift/message-board/message-service/src/test/java/org/hibernate/demo/message/post/it/MessageTaskIT.java]: Can we read the version from somewhere? I think in OGM we have something for this

Comment (yrodiere) [reply openshift/message-board/message-service/src/test/java/org/hibernate/demo/message/post/it/MessageTaskIT.java]: In Search we just use a resource file, configure Maven to filter it, and read the file when executing tests.

See:

* https://github.com/hibernate/hibernate-search/blob/master/legacy/integrationtest/wildfly/src/test/resources/module-versions.properties
* https://github.com/hibernate/hibernate-search/blob/master/legacy/integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/VersionTestHelper.java#L113
* https://github.com/hibernate/hibernate-search/blob/master/legacy/integrationtest/wildfly/pom.xml#L229

Comment (fax4ever) [reply openshift/message-board/message-service/src/test/java/org/hibernate/demo/message/post/it/MessageTaskIT.java]: You're right @DavideD and thank you @yrodiere for the suggestion, it makes sense for me. I applied the same strategy here https://github.com/infinispan/infinispan/pull/6175.

**Comment by fax4ever** [comment]
Merged #33 Thanks

**Pull request merged**